tests/parse_metadata.py

```diff
@@ -30,6 +32,8 @@
     "read_stubtest_settings",
 ]
 
+OLDEST_SUPPORTED_PYTHON: Final = "3.7"
+PYTHON_VERSION: Final = f"{sys.version_info.major}.{sys.version_info.minor}"
```

Comment:
Hmm... on second thoughts... maybe this constant really belongs in `utils.py`, rather than `parse_metadata.py`? This file already depends on `utils.py`, so I don't think importing it from `utils.py` would be a problem